qiskit/backends/ibmq/ibmqjob.py

```diff
@@ -129,14 +130,20 @@ def __init__(self, api, is_device, qobj=None, job_id=None, backend_name=None,
             backend_name(String): The name of the backend that run the job.
             creation_date(String): When the job was run.
 
+        Raises:
+            QISKitError: if the input Qobj is invalid
```

Comment:
This one is probably a `QobjValidationError`? For consistency, can you also either add a similar `Raises` in the docstring of the corresponding function of `LocalJob` - or remove both of them, as we don't really enforce declaring the errors that are raised in children functions (maybe we should at some point and in some cases such as this one, though, as it can be quite handy).

Reply:
I'm going to remove them since they should be raised in the `submit` functions but leave the proper notes to do it.